models/stg_linkedin__ad_analytics_by_creative.sql

```diff
@@ -26,6 +26,11 @@ with base as (
         {% else %}
         cost_in_usd as cost
         {% endif %}
+
+        {% if var('linkedin__passthrough_metrics', []) != [] %}
+        , {{ var('linkedin__passthrough_metrics', [] )  | join(', ') }}
+        {% endif %}
+
```

Comment:
Don't we also need to add the conditional to the `macro` cte at line [14](https://github.com/fivetran/dbt_linkedin_source/pull/18/files#diff-e50f115854b2c57b89ff0c253f4547e69a564a1816916a5cbdd9872b6d67c871R14)?

Reply:
i did originally but that threw an error, since the staging column macro for this table actually includes the metrics that folks were talking about (ie video_views). so this variable passes through non-custom metrics that we exclude in the `fields` CTE of this model. maybe the readme should reflect that more clearly...

Reply:
Ah that makes sense, thanks!